arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Anytrust orbit chains keyset generation'
+sidebar_label: 'Keyset Generation for Anytrust Orbits'
+description: 'Keyset generation and setup for Anytrust Orbit chains'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Anytrust Orbit chains.'
+sidebar_position: 1
+---
+**Anytrust** chains, relies on an external Data Availability Committee (hereafter, "the Committee") to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
+
+The Committee comprises N members, among whom it is presumed by AnyTrust that a minimum of H members maintain integrity. H is the minimum number of trusted committee members on Anytrust chains, and it is configurable by the "assumed-honest" parameter in the keyset, and by chain owner(s). Consequently, in scenarios where `K = (N + 1) - H` members of the Committee pledge to grant access to specific data, and they must sign and attest they have the data for store to be considered successful.
```

Comment:
```suggestion
The DAC has _N_ members; the AnyTrust protocol assumes that a minimum of _H_ DAC members maintain integrity. _H_ is the minimum number of trusted committee members on AnyTrust chains, configurable by the chain's owner via the `assumed-honest` parameter in the keyset. In scenarios where `K = (N + 1) - H` members of the DAC pledge to grant access to specific data, they must sign and attest they have the data for store to be considered successful.
```

Does this edit feel like an improvement?

> Consequently, in scenarios where `K = (N + 1) - H` members of the Committee pledge to grant access to specific data, and they must sign and attest they have the data for store to be considered successful.

I don't understand what we're trying to communicate here. Would you mind rearticulating / clarifying?